spec/controllers/just_so_you_know_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe EligibleController do

```

Comment:
This should be `JustSoYouKnowController`

Reply:
Yes, yes it should. Changes added to PR.